src/panels/Scene/hooks.ts

```diff
@@ -0,0 +1,60 @@
+import { BoxData, PanelData, TabData } from 'rc-dock';
+
+import { useWindowLayoutProvider } from '@/windowmanagement/WindowLayout/hooks';
+
+export const DefaultSgnWindowId = 'defaultSceneGraphNodeWindow';
+
+export function useOpenCurrentSceneNodeWindow() {
+  const { ref, addWindow, closeWindow, createWindowTabData } = useWindowLayoutProvider();
+
+  function openCurrentNodeWindow(content: JSX.Element) {
+    if (!ref || !('current' in ref) || ref.current === null) {
+      throw new Error('WindowLayoutProvider ref is not set');
+    }
+
+    const existingWindow = ref.current.find(DefaultSgnWindowId);
+    if (!existingWindow) {
+      addWindow(content, {
+        id: DefaultSgnWindowId,
+        title: 'Scene: Selected Node',
+        position: 'float'
+      });
+
+      const newWindow = ref.current.find(DefaultSgnWindowId) as TabData | PanelData;
+      const scenePanelParentBox = ref.current.find('scene')?.parent as BoxData;
+
+      if (!newWindow || !scenePanelParentBox) {
+        throw new Error('Could not find the new window or the scene panel parent box');
+      }
+
+      if (scenePanelParentBox.parent && scenePanelParentBox.parent.mode === 'float') {
+        // @TODO (emmbr, 2024-12-04): If floating window, split the scene tree in two
+        // instead?
```

Comment:
I don't think we want to do this, due to added complexity. So remove this comment. 

It now pops up as another floating window, which is good enough